pom.xml

```diff
@@ -19,24 +32,11 @@
     </license>
   </licenses>
 
-  <organization>
-    <name>doubleSlash Net-Business GmbH</name>
-    <url>https://www.doubleslash.de/</url>
-  </organization>
-
-  <parent>
-    <groupId>org.springframework.boot</groupId>
-    <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.7.5</version>
-    <relativePath /> <!-- lookup parent from repository -->
-  </parent>
-
   <properties>
-    <project.version>1.3.0-SNAPSHOT</project.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.compiler.source>11</maven.compiler.source>
-    <maven.compiler.target>11</maven.compiler.target>
+    <java.version>17</java.version>
+    <javafx.version>20.0.1</javafx.version>
```

Comment:
currently just to don't forget: "JavaFX 20 Requires JDK 17 or Later"
https://github.com/openjdk/jfx/blob/master/doc-files/release-notes-20.md#javafx-20-requires-jdk-17-or-later

If we wanted to play around with compile target version the javafx dependency would require 17 at least